src/NServiceBus.Transport.RabbitMQ/Administration/ManagementApi/ManagementClient.cs

```diff
@@ -11,7 +11,7 @@ namespace NServiceBus.Transport.RabbitMQ.ManagementApi;
 using System.Threading;
 using System.Threading.Tasks;
 
-class ManagementClient : IDisposable
+sealed class ManagementClient : IDisposable
```

Comment:
Do we need the class to be sealed?

Reply:
I went ahead and reverted this change along with some other tweaks.